Content/StartService.md

```diff
@@ -0,0 +1,17 @@
+# Start Service
+
+_StartService_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). This feature provides an automatic start for the specified service. Any specified service will be started automatically. For example:
```

Comment:
```suggestion
_StartService_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). This feature provides an automatic start for the specified service when installed outside the sandbox. Any specified service will be started automatically in the sandbox. For example:
```

This might also apply to StartProgram, a further clarification from @DavidXanatos would be appreciated.